javascriptv3/example_code/medical-imaging/actions/search-image-sets.js

```diff
@@ -0,0 +1,110 @@
+    //         extendedRequestId: undefined,
+    //         cfId: undefined,
+    //         attempts: 1,
+    //         totalRetryDelay: 0
+    //     },
+    //     imageSetsMetadataSummaries: [
+    //         {
+    //             DICOMTags: [Object],
+    //             createdAt: "2023-09-19T16:59:40.551Z",
+    //             imageSetId: '7f75e1b5c0f40eac2b24cf712f485f50',
+    //             updatedAt: "2023-09-19T16:59:40.551Z",
+    //             version: 1
+    //         }]
+    // }
+
+    return response["imageSetsMetadataSummaries"];
+};
+// snippet-end:[medical-imaging.JavaScript.resource.searchImageSetV3]
+
+// Invoke the following code if this file is being run directly.
+if (process.argv[1] === fileURLToPath(import.meta.url)) {
+    // snippet-start:[medical-imaging.JavaScript.resource.searchImageSetV3.datastoreID]
+    const datastoreId = "12345678901234567890123456789012";
+    // snippet-end:[medical-imaging.JavaScript.resource.searchImageSetV3.datastoreID]
+    // Search using EQUAL operator.
+    // snippet-start:[medical-imaging.JavaScript.resource.searchImageSetV3.equalFilter]
+    try {
+        const filters = [{
+            values: [{DICOMPatientId: "9227465"}],
+            operator: "EQUAL"
+        }];
+
+        await searchImageSets(datastoreId, filters);
+    } catch (err) {
+        console.error(err);
+    }
+    // snippet-end:[medical-imaging.JavaScript.resource.searchImageSetV3.equalFilter]
+
+    // Search with BETWEEN operator using DICOMStudyDate and DICOMStudyTime.
+    // snippet-start:[medical-imaging.JavaScript.resource.searchImageSetV3.betweenFilter1]
+    try {
+        const filters = [{
+            values: [{
+                DICOMStudyDateAndTime: {
+                    DICOMStudyDate: '19900101',
+                    DICOMStudyTime: '000000'
+                }
+            },
+                {
+                    DICOMStudyDateAndTime: {
+                        DICOMStudyDate: '20230901',
+                        DICOMStudyTime: '000000'
+                    }
+                }],
+            operator: "BETWEEN"
+        }];
+
+        await searchImageSets(datastoreId, filters);
+    } catch (err) {
+        console.error(err);
+    }
+    // snippet-end:[medical-imaging.JavaScript.resource.searchImageSetV3.betweenFilter1]
+
+    // Search with BETWEEN operator and createdAt date.
+    // snippet-start:[medical-imaging.JavaScript.resource.searchImageSetV3.betweenFilter2]
+    try {
+        const filters = [{
+            values: [
+                {createdAt: new Date("1985-04-12T23:20:50.52Z")},
+                {createdAt: new Date("2023-09-12T23:20:50.52Z")},
+            ],
+            operator: "BETWEEN"
+        }];
+
+        await searchImageSets(datastoreId, filters);
+    } catch (err) {
+        console.error(err);
+    }
+    // snippet-end:[medical-imaging.JavaScript.resource.searchImageSetV3.betweenFilter2]
+}
+``
```

Comment:
Errant backticks?

Reply:
surprised that did not show up on my webstorm problems pane.